lectures/physics_2/SDLProject/Entity.h

```diff
@@ -1,88 +1,93 @@
-enum EntityType { PLATFORM, PLAYER, ITEM };
-
 class Entity
 {
 private:
     bool m_is_active = true;
-    
+
     // ––––– ANIMATION ––––– //
-    int *m_animation_right = NULL; // move to the right
-    int *m_animation_left  = NULL; // move to the left
-    int *m_animation_up    = NULL; // move upwards
-    int *m_animation_down  = NULL; // move downwards
-    
+    int*  m_animation_right = NULL, // move to the right
+        * m_animation_left  = NULL, // move to the left
+        * m_animation_up    = NULL, // move upwards
+        * m_animation_down  = NULL; // move downwards
+
     // ––––– PHYSICS (GRAVITY) ––––– //
     glm::vec3 m_position;
     glm::vec3 m_velocity;
     glm::vec3 m_acceleration;
-    
-    float m_width  = 1;
+
+    float m_width = 1;
     float m_height = 1;
-    
+
+
 public:
-    // ––––– STATIC ATTRIBUTES ––––– //
+    // ————— STATIC VARIABLES ————— //
     static const int SECONDS_PER_FRAME = 4;
-    static const int LEFT  = 0,
-                     RIGHT = 1,
-                     UP    = 2,
-                     DOWN  = 3;
-    
-    // ––––– SETUP AND RENDERING ––––– //
-    GLuint m_texture_id;
-    glm::mat4 m_model_matrix;
-    EntityType m_type;
-    
-    // ––––– TRANSLATIONS ––––– //
-    float m_speed;
-    glm::vec3 m_movement;
-    
-    // ––––– ANIMATIONS ––––– //
-    int **m_walking          = new int*[4] { m_animation_left, m_animation_right, m_animation_up, m_animation_down };
-    int *m_animation_indices = NULL;
-    int m_animation_frames   = 0;
-    int m_animation_index    = 0;
+    static const int LEFT   = 0,
+                     RIGHT  = 1,
+                     UP     = 2,
+                     DOWN   = 3;
+
+    // ————— ANIMATION ————— //
+    int** m_walking = new int* [4]
+        {
+            m_animation_left,
+            m_animation_right,
+            m_animation_up,
+            m_animation_down
+        };
+
+    int m_animation_frames = 0,
+        m_animation_index  = 0,
+        m_animation_cols   = 0,
+        m_animation_rows   = 0;
+
+    int* m_animation_indices = NULL;
     float m_animation_time   = 0.0f;
-    int m_animation_cols     = 0;
-    int m_animation_rows     = 0;
-    
+
     // ––––– PHYSICS (JUMPING) ––––– //
-    bool m_is_jumping     = false;
-    float m_jumping_power = 0;
-    
+    bool  m_is_jumping     = false;
+    float m_jumping_power  = 0;
+
     // ––––– PHYSICS (COLLISIONS) ––––– //
     bool m_collided_top    = false;
     bool m_collided_bottom = false;
     bool m_collided_left   = false;
     bool m_collided_right  = false;
 
-    // ––––– METHODS ––––– //
+    // ————— TRANSFORMATIONS ————— //
+    float     m_speed;
+    glm::vec3 m_movement;
```

Comment:
Same thing here as with my comment on physics_2's review request. m_movement was made private in the entity lecture.